plugins/inputs/windows_event_log/windows_event_log.go

```diff
@@ -77,6 +81,15 @@ func (s *Plugin) FindLogSrc() []logs.LogSrc {
  * We can do any initialization in this method.
  */
 func (s *Plugin) Start(acc telegraf.Accumulator) error {
+	alreadyRan := true
+	startOnlyOnce.Do(func() {
+		fmt.Println("Warning! This function can only be executed once, this is the first and last time it will be ran.")
```

Comment:
Do not use `fmt`. Use `log` for log lines. I don't think we want to have these logged anyways. It's understood that this will be called multiple times in some scenarios. We don't want to mislead customers into thinking this is an issue.

Reply:
should we just ignore if they run it twice?